CHANGELOG.md

```diff
@@ -204,6 +204,59 @@
   deprecated environment variables.
   ([Surya Rose](https://github.com/GearsDatapacks))
 
+- When generating documentation, the build tool now prints correct type variable
+  names for function signatures. For example, this code:
```

Comment:
They were correct before! The change is that it uses the same type variable names that were used in the code.